packages/office-ui-fabric-react/src/components/ComboBox/ComboBoxPage.tsx

```diff
@@ -0,0 +1,59 @@
+import * as React from 'react';
+import {
+  ExampleCard,
+  IComponentDemoPageProps,
+  ComponentPage,
+  PropertiesTableSet
+} from '@uifabric/example-app-base';
+import { ComboBoxBasicExample } from './examples/ComboBox.Basic.Example';
+
+const ComboBoxBasicExampleCode = require('!raw-loader!office-ui-fabric-react/src/components/ComboBox/examples/ComboBox.Basic.Example.tsx') as string;
+
+export class ComboBoxPage extends React.Component<IComponentDemoPageProps, {}> {
+  public render() {
+    return (
+      <ComponentPage
+        title='ComboBox'
+        componentName='ComboBoxExample'
+        exampleCards={
+          <ExampleCard title='ComboBox' code={ ComboBoxBasicExampleCode }>
+            <ComboBoxBasicExample />
+          </ExampleCard>
+        }
+        propertiesTables={
+          <PropertiesTableSet
+            sources={ [
+              require<string>('!raw-loader!office-ui-fabric-react/src/components/ComboBox/ComboBox.Props.ts')
+            ] }
+          />
+        }
+        overview={
+          <div>
+            <p>
+              A ComboBox is a list in which the selected item is always visible, and the others are visible on demand by clicking a drop-down button or by typing in the input (unless allowFreeform and autoComplete are both false). They are used to simplify the design and make a choice within the UI. When closed, only the selected item is visible. When users click the drop-down button, all the options become visible. To change the value, users open the list and click another value or use the arrow keys (up and down) to select a new value. When collapsed if autoComplete and/or allowFreeform are true, the user can select a new value by typing.

```

Comment:
Dorpdown allows the user to specify an OnRenderTitle which returns a JSXElement. This works fine for the dropdown case since be default it just uses a span to hold the text, but a combobox uses an input to align with the correct accessibility representation. After that there's a lot of other minor differences around changing the selected option. I feel like eventually the comboBox can replace the dropdown but that initially both would exist and the dropdown could be marked up as deprecated. Note the details in the main description